### ISSUE TITLE
Support cone-beam geometry given by a set of projection matrices

### ISSUE THREAD

**Issue by wuerflts** [ner0-m/curadon#1]
I did not find a possibility to use a set of projections with associated projection matrices for the cone-beam case. As this is the most common representation of acquisition geometry it would be of utmost importance to support it.

**Comment by ner0-m** [comment]
Hi! You are correct, currently, that is not supported. I'm currently trying out a couple of things in the 2D case, once that is done, I'll shift my focus on 3D again and that will be one of the first things I'll do. I'll update here once that is done!

Thanks for the interest, I appreciate it!

**Comment by wuerflts** [comment]
Yes I really appreciate your project. I once drafted the same library with the exact same name since I believe this is the only way we can have a reusable implementation for all frameworks, if we do it with a C interface as low-level library in the spirit of cudnn. Never came round to doing it so far.
